src/test/java/steps/AddressSteps.java

```diff
@@ -1,9 +1,26 @@
+/*
+ * @(#) FormPageFour.java Copyright (c) 2019 Jala Foundation.
```

Comment:
change the name of FormPageFour.java accord to your classes

Reply:
Solved.